recipes/slackbot/LICENSE.txt

```diff
@@ -0,0 +1,7 @@
+Copyright (C) 2015 Slackbot Contributors
```

Comment:
Please make a PR upstream to include this in the MANIFEST.in file of the upstream sdist.

Reply:
PR is here: https://github.com/lins05/slackbot/pull/200

But the project is not very active. Could we merge this and I'll update the feedstock once the PR is merged?

Reply:
Yup! We should simply attempt to submit a PR. That is all.

Reply:
Perfect. Thanks.